nuxeo-connect-client/src/main/java/org/nuxeo/connect/registration/RegistrationHelper.java

```diff
@@ -160,7 +165,8 @@ public static String remoteRegisterInstance(String login, String password, Strin
         } catch (UnsupportedEncodingException e) {
             throw new RuntimeException(e);
         }
-        try (CloseableHttpClient httpClient = HttpClients.createDefault();
+        HttpClientBuilder httpClientBuilder = getHttpClientBuilder(null, null, url);
+        try (CloseableHttpClient httpClient = httpClientBuilder.build();
```

Comment:
```suggestion
        try (CloseableHttpClient httpClient = getHttpClientBuilder(null, null, url).build();
```
To be consistent maybe :D

Reply:
Ah yes, I missed that